src/components/Request/NewFormSection/NewForm.LinkedInPost.tsx

```diff
@@ -0,0 +1,36 @@
+import * as FormFields from "components/Request/FormFields/FormFields";
+import { Divider, InfoLabel, Title2 } from "@fluentui/react-components";
+import { useFormContext } from "react-hook-form";
+
+const LinkedInPost = () => {
+  const form = useFormContext();
+  const linkedinQualifications = form.watch("linkedinQualifications");
```

Comment:
You should leverage useWatch instead.  You don't need to get the form context since it is nested inside a form.  With useWatch, you can specify which fields to watch and it will only re-render if those fields change.  With form context it re-renders with every change of the form.